org.eclipse.m2e.tests/src/org/eclipse/m2e/tests/archetype/ArchetypeManagerTest.java

```diff
@@ -66,8 +65,6 @@ public void setUp() throws Exception {
   @Test
   public void testArchetypeManager() throws Exception {
     {
-      ArchetypeCatalogFactory catalog = archetypeManager.getArchetypeCatalogFactory(NexusIndexerCatalogFactory.ID);
-      assertNotNull(catalog);
       ArchetypeCatalog archetypeCatalog = catalog.getArchetypeCatalog();
```

Comment:
This change causes a compiler error since the variable `catalog` is not defined anymore. Therefore the builds at m2e-core fail currently.

Reply:
Should the entire content within the braces be removed?